### DISCUSSION TITLE
Internet connection is required Steam lan server

### DISCUSSION THREAD

**Discussion by elyashivhazan** [mbround18/valheim-docker#948] [Help!] [answered]
hi the new version requires the server to have internet connection what settings to use to have the server back in the steam game server browser for lan only use?

**Comment by mbround18** [comment]
I think you would need public 0 and to connect with a lan IP to reach the server locally. Ill have to redownload and try with the new version or see if they added any new flags that might facilitate local play better

**Reply by elyashivhazan** [reply]
i set public to 0 and crosplay is disabled but still the server must connect to the internet if it dosnt it says
```
...
Am I Host? True
-------------------: Game server connected failed
Unloading 2 unused Assets to reduce memory usage. Loaded Objects now: 195123.
Total: 70.235570 ms (FindLiveObjects: 9.638012 ms CreateObjectMapping: 10.968976 ms MarkObjects: 49.404154 ms  DeleteObjects: 0.223527 ms)
-------------------: Game server connected failed
-------------------: Game server connected failed
-------------------: Game server connected failed
```

**Reply by mbround18** [reply]
@elyashivhazan this might be better directed to https://discord.com/invite/valheim

**Reply by elyashivhazan** [reply]
hi i have all ready manage to fix that, thx.

**Reply by mbround18** [reply]
Hey @elyashivhazan how did you solve it? so that in case others want to replicate they can

**Reply by elyashivhazan** [reply, marked as answer]
i used a steam emu to bypass steam's requirements for server connectivity.
